cypress/pageobjects/template.po.ts

```diff
@@ -38,4 +35,12 @@ export default class TemplatePage extends CruResourcePo {
   memory() {
     return new LabeledInputPo('.labeled-input', `:contains("Memory")`)
   }
+
+  save(namespace:string, createButton: string = 'Create') {
+    cy.intercept('POST', `/v1/harvester/${this.realType}s/${namespace}`).as('create');
+    cy.get(this.footerButtons).contains(createButton).click()
+    cy.wait('@create').then(res => {
+      expect(res.response?.statusCode, `Create ${this.type} success`).to.be.oneOf([200, 201]);
+    })
+  }
```

Comment:
Is it still necessary? I think it's in `cru-resource` class

Reply:
There is a parameter named createButton in the function `save`.
creating new template version is contrary to creating other CRD. Craete-button text on creating template version page is `Save` not `Create`.